### ISSUE TITLE
Bomb IconEntities too high

### ISSUE THREAD

**Issue by turbopasi** [kaloczikvn/kPM#47]
Mostly on smaller maps with multiple levels/floors it can be really confusing when the bomb marker icon is not actually at the bombsite, but around 10 meters above. 

We are running promod currently on Scrapmetal , take a look at the screenshot : 
![marker](https://i.imgur.com/jGbBVlC.jpg)

Is this by design ? Or just never noticed ? 

It's probably something small around those lines : 
```lua
function kPMShared:SpawnIconEntities(p_Trans, p_Id)
    local s_CustomMapMarkerEntityData = nil
    print("SpawnIconEntities: "..p_Id)
    if p_Id == "A" then
        s_CustomMapMarkerEntityData = MapMarkerEntityData(ResourceManager:SearchForInstanceByGuid(self.m_CustomMapMarkerEntityAGuid))
    elseif p_Id == "B" then
        s_CustomMapMarkerEntityData = MapMarkerEntityData(ResourceManager:SearchForInstanceByGuid(self.m_CustomMapMarkerEntityBGuid))
    end

    if s_CustomMapMarkerEntityData ~= nil then
        local s_EntityPos = LinearTransform()
        s_EntityPos.trans = p_Trans.trans

        local s_CreatedEntity = EntityManager:CreateEntity(s_CustomMapMarkerEntityData, s_EntityPos)

        if s_CreatedEntity ~= nil then
            s_CreatedEntity:Init(Realm.Realm_ClientAndServer, true)
        end
    else
        print('err: s_CustomMapMarkerEntityData - could not spawn icon.')
    end
end
```

**Comment by kaloczikvn** [comment]
`ext/Shared/MapMarkerEntityDataGenerator.lua - Line 46.`

You could try tweaking `Vec3(0, 6.4, 0)` but the ultimate solution is to replace these markers with mission markers, but i haven't succeded in it yet.

**Comment by kaloczikvn** [comment]
Fixed in 7edf1bfd59e0f6fdfa2839fcc9c88e944f4fd8a2
